src/generate/azgenerator.ts

```diff
@@ -51,6 +51,7 @@ export async function processRequest(host: Host) {
             }
         }
 
+        const tolintFiles = [];
         for (const f in files) {
             if (!isNullOrUndefined(files[f])) {
```

Comment:
why this would happen?

Reply:
This was originally added to fix an issue from application-insight https://github.com/Azure/autorest.az/pull/589/files, but I just tried again, it still works if we remove the if logic, guess it has been considered somewhere else about this conner case.

Reply:
so I removed it.